### PR TITLE
Wire session recognition with website API

### PR DESCRIPTION
**Pull request by erickpintor**
Closes: https://github.com/faunadb/dashboard/issues/44

I think there is a lot of room for improvements in this code but, I'm trying to do just the minimum required for the next release.

### CODE REVIEW

**Comment by jchris** [comment]
looks good to me. We should plan to make the useExternalSignin() return a type of external signin (maybe something that resolves to a plugin name) and then have one plugin be FaunaCloud.

**Comment by erickpintor** [comment]
Yeah. Once we settle down on what is our modularization strategy, we should totally move all cloud related workflow to somewhere else.

File: package.json [hunk 1]
```diff
@@ -19,7 +20,8 @@
     "react-split-pane": "^0.1.57",
     "react-treebeard": "^1.1.4",
     "redux": "^3.6.0",
-    "redux-thunk": "^2.1.0"
+    "redux-thunk": "^2.1.0",
+    "superagent": "^3.3.2"
```
Comment (marrony) [inline package.json]: I see our fauna client uses a very different version of superagent, would that be a problem?

```
npm install
console@0.1.0 dashboard
├─┬ faunadb@0.2.2
│ └── superagent@1.8.5
├── js-cookie@2.1.3
└─┬ superagent@3.3.2
  ├─┬ form-data@2.1.2
  │ ├── asynckit@0.4.0
  │ └─┬ mime-types@2.1.14
  │   └── mime-db@1.26.0
  ├── qs@6.3.0
  └─┬ readable-stream@2.2.2
    └── isarray@1.0.0
```

Comment (marrony) [inline package.json]: I see our fauna client uses a very different version of superagent, would that be a problem?

```
npm install
console@0.1.0 dashboard
├─┬ faunadb@0.2.2
│ └── superagent@1.8.5
├── js-cookie@2.1.3
└─┬ superagent@3.3.2
  ├─┬ form-data@2.1.2
  │ ├── asynckit@0.4.0
  │ └─┬ mime-types@2.1.14
  │   └── mime-db@1.26.0
  ├── qs@6.3.0
  └─┬ readable-stream@2.2.2
    └── isarray@1.0.0
```

Comment (erickpintor) [inline package.json]: I don't think it will but I've downgraded the dashboard's version just to be safe.

**Pull request merged**